src/sentry/incidents/subscription_processor.py

```diff
@@ -633,19 +635,65 @@ def trigger_resolve_threshold(
     def handle_trigger_actions(
         self, incident_triggers: List[IncidentTrigger], metric_value: float
     ) -> None:
-        actions = deduplicate_trigger_actions(triggers=deepcopy(incident_triggers))
+        actions = deduplicate_trigger_actions(triggers=deepcopy(self.triggers))
```

Comment:
Big change here was that instead of getting only the actions for the `incident_triggers`, we get all actions associated with any trigger that belongs to this alert rule.